### PR TITLE
Diona Friendly Ninja Suit

### PR DESCRIPTION
**Pull request by DieselMohawk**
## About the PR
 Removed the movement changes from the ninja shoes and instead put them on the ninja suit.

## Why / Balance
 Since ninjas can now be any species the role needs to support all species. Dionae cannot wear shoes so I wanted to allow them to still be fast.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

🆑 
-tweak: Made Ninja Suit Diona friendly.

### CODE REVIEW

**Comment by Cojoke-dot** [comment]
Put the noslip on the suit too, rolling diona as ninja just to be slipped would feel terrible

**Comment by deltanedas** [comment]
add magboots to engi hardsuit so diona engi can do whatever the fuck engi needs magboots for: :trollface:

**Comment by DieselMohawk** [comment]
> Put the noslip on the suit too, rolling diona as ninja just to be slipped would feel terrible
> 
> The suit is also meant to slow you down by 10%, may wanna drop the speed values to account for that. image

I'm gonna leave of the no-slips so the Diona species downside is still there, they could always teleport over the puddles. I'm looking to find where the -10% movement speed comes from.

**Comment by Plykiya** [comment]
probably inherited from ClothingOuterBaseLarge

**Comment by DieselMohawk** [comment]
> probably inherited from ClothingOuterBaseLarge

Yeah, I found it. Gonna do that real quick.

**Comment by DieselMohawk** [comment]
Made it the proper speed now.

**Comment by DieselMohawk** [comment]
And no, I didn't not change the component itself, I just used a different parent and made the item bigger. I don't fuck with components.

File: Resources/Prototypes/Entities/Clothing/OuterClothing/suits.yml [hunk 1]
```diff
@@ -149,6 +151,9 @@
         Slash: 0.8
         Piercing: 0.8
         Heat: 0.8
+  - type: ClothingSpeedModifier
```
Comment (Cojoke-dot) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/suits.yml:154]: You want to change this component, not the parent. Try this instead and reverse the other stuff you did:
  - type: ClothingSpeedModifier
    sprintModifier: 1.2

Comment (DieselMohawk) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/suits.yml:154]: Sure

Comment (Cojoke-dot) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/suits.yml:152]: (also remove the walk speed modifier)

Comment (DieselMohawk) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/suits.yml:152]: Guh

**Comment by DieselMohawk** [comment]
This made me learn what conflict markers are.

**Comment by UbaserB** [comment]
After internal maintainer discussion, this PR has been decided to be closed. This is because it is a little weird for the suit to improve your speed and while being unable to wear shoes is a huge combat debuff for diona in general, but I don't think this is a problem for ninjas since they are a ghost role. They will just play a little different sometimes, there is no real need to balance for metagaming reasons (compared to nukies for example).

(we forgot to add the tag for discussion, sorry about that, but for reference the discussion started on 08/26)

**Pull request closed without merging**